src/Repository/YesplanEventRepository.php

```diff
@@ -70,18 +70,20 @@ public function findOldEvents(): array
      */
     public function findNewProductionOnlineEvents(): array
     {
-        //get all events with productiononline = 1 not already created in Asana
+        // get all events with productiononline = 1 not already created in Asana
```

Comment:
This comment (on the return value) does not match the one in the PHPDoc block for the function. Update the PHPDoc block and clean up this comment.